### PR TITLE
[FIX] l10n_ar_purchase: Cambiamos los nombres de las fechas en los reportes de compra

### PR DESCRIPTION
**Pull request by mem-adhoc**
Ticket: 76148

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/890).

**Comment by pablohmontenegro** [comment]
@mem-adhoc can you rebase please?

**Comment by mem-adhoc** [comment]
@pablohmontenegro Pablito, I made the rebase already!

**Comment by pablohmontenegro** [comment]
@roboadhoc r+

**Comment by roboadhoc** [comment]
@pablohmontenegro you may want to rebuild or fix this PR as it has failed CI.

File: l10n_ar_purchase/views/purchase_report_templates.xml [hunk 1]
```diff
@@ -13,12 +13,28 @@
...
+    </template>
+
+    <template id="custom_header_purchaseorder" inherit_id="l10n_ar.custom_header" primary="True" priority="99">
+        <xpath expr="//div[hasclass('col-6', 'text-end')]/span[1]" position="replace">
+            <t t-if="o.state in ['purchase', 'done']">
+                <span t-att-style="'color: %s;' % o.company_id.secondary_color">Approve date: </span>
+            </t>
+            <t t-else="">
+                <span t-att-style="'color: %s;' % o.company_id.secondary_color">Date due: </span>
+            </t>
```
Comment (zaoral) [inline l10n_ar_purchase/views/purchase_report_templates.xml:29]:
```suggestion
            <span t-att-style="'color: %s;' % o.company_id.secondary_color"> 
               <t t-if="o.state in ['purchase', 'done']"> Approve date: </t>
               <t t-else=""> Date due: </t>
            </span>
```

File: l10n_ar_purchase/views/purchase_report_templates.xml [hunk 2]
```diff
@@ -13,12 +13,28 @@
         </t>
     </template>
 
+    <template id="custom_header_purchasequotation" inherit_id="l10n_ar.custom_header" primary="True" priority="99">
+        <xpath expr="//div[hasclass('col-6', 'text-end')]/span[1]" position="replace">
+            <span t-att-style="'color: %s;' % o.company_id.secondary_color">Date due: </span>
+        </xpath>
+    </template>
+
+    <template id="custom_header_purchaseorder" inherit_id="l10n_ar.custom_header" primary="True" priority="99">
```
Comment (zaoral) [inline l10n_ar_purchase/views/purchase_report_templates.xml]: do we need priorty 99?

**Comment by zaoral** [comment]
@roboadhoc r+

**Pull request closed without merging**